.editorconfig

```diff
@@ -13,6 +13,9 @@ insert_final_newline = true
 [*.{java,kt,kts,scala,rs,xml,kt.spec,kts.spec}]
 indent_size = 4
 
+[*.{kt,kts}]
+kotlin_imports_layout=ascii
```

Comment:
I'm not sure if we should use "idea" for ktlint itself.. If yes, then I'd prefer to do it in a separate PR afterwards, but can also do it here

Reply:
I personally use IDEA but I rarely use the IDE's code formatting so I don't have a strong opinion.

Reply:
I would say it is fine. And this property could be in the future not really IDEA specific.